aten/src/ATen/native/cudnn/Conv_v8.cpp

```diff
@@ -76,99 +139,533 @@ void filterEngineConfigs(
 
 struct CacheKey {
   ConvolutionParams params;
-  uint8_t input_alignment;
-  uint8_t weight_alignment;
-  uint8_t output_alignment;
+  cudnnBackendDescriptorType_t operation;
```

Comment:
what are possible values of `operation`?

Reply:
seems to be a one-size-fits-all enum... https://docs.nvidia.com/deeplearning/cudnn/api/index.html#cudnnBackendDescriptorType_t